### PR TITLE
Drake version

### PR DESCRIPTION
**Pull request by RussTedrake**
add drake_version.json and use it in the text and in CircleCI.

### CODE REVIEW

**Comment by RussTedrake** [comment]
fyi @gizatt @peteflorence @EricCousineau-TRI .

**Comment by peteflorence** [comment]
Very good!

**Comment by peteflorence** [comment]
@RussTedrake I just want to praise the existence of the `drake_version.json` -- very much helps!

File: underactuated.html [hunk 1]
```diff
@@ -3878,15 +3898,16 @@ <h1>Acrobots, Cart-Poles, and Quadrotors</h1>
       we see that this function might need to be non-smooth for even very simple
       dynamics and objectives.</p>
 
-      <p>One natural way to parameterize $\hat{J}^*$ -- a scalar valued-function
+      <>One natural way to parameterize $\hat{J}^*$ -- a scalar valued-function
```
Comment (EricCousineau-TRI) [inline underactuated.html]: Sorry for the late follow-up - is this a DOM Error? (should be `<p>`?)

**Comment by EricCousineau-TRI** [comment]
Awesome!

**Pull request merged**